core/integration/client_tests.go

```diff
@@ -67,23 +67,34 @@ LOA+tLe/MbwZ69SRdG6Rx92f9tbC6dz7UVsyI7vIjS+961sELA6FeR91lA==
 -----END PUBLIC KEY-----`
 )
 
+func runBatchAndPublish(ctx context.Context, dirID string, mn, mx int32, block bool, env *Env) error {
+	rbReq := &spb.RunBatchRequest{
+		DirectoryId: dirID,
+		MinBatch:    mn,
+		MaxBatch:    mx,
+	}
+	convert := func(err error, prefix string) error {
```

Comment:
Does https://godoc.org/google.golang.org/grpc/status#FromContextError help?

Reply:
Nah, direct checks seem more convenient. Do you see a better way to express this?